modules/codefresh/Makefile.pipeline

```diff
@@ -0,0 +1,65 @@
+## To get GIT_* env variables reference https://github.com/cloudposse/build-harness/blob/master/modules/git/bootstrap.Makefile
+
+# Extract ticket number from branch name (e.g. feature/ch37684/add-codefresh)
+FEATURE ?= $(shell echo "$(GIT_BRANCH)" | cut -s -d'/' -f2 | tr '[:upper:]' '[:lower:]')
+
+PIPELINE_ACTION = pass
+
+## Pre-production deploys only released versions
+ifeq ($(PIPELINE_ENV),pre-production)
+
+  NAMESPACE ?= $(PROJECT)-pre-prod
```

Comment:
I think we should move `NAMESPACE`, `APP_HOST` and `RELEASE_NAME` out of this conditional logic.